src/pages/questionPage.js

```diff
@@ -50,18 +52,20 @@ export const initQuestionPage = () => {
 const nextQuestion = () => {
   const correctAnswer = quizData.questions[quizData.currentQuestionIndex].correct;
   const addClass = quizData.currentQuestionAnswer === correctAnswer ? 'correct' : 'wrong';
-  const body = document.getElementById(USER_INTERFACE_ID);
+  const answerBody = quizData.currentAnswerElement;
```

Comment:
@beyzagoknl Since `currentAnswerElement` isn't shared between multiple pages, and also isn't needed in any other part of the application, you can just define a `let` variable in the top of this file, and assign it to that variable.

Reply:
The purpose of quizData data object is more to hold information (text, numbers, booleans), not really to hold values of HTML elements.